test/utils/image/manifest.go

```diff
@@ -252,7 +252,7 @@ func initImageConfigs() map[int]Config {
 	configs[Porter] = Config{e2eRegistry, "porter", "1.0"}
 	configs[PrometheusDummyExporter] = Config{e2eRegistry, "prometheus-dummy-exporter", "v0.1.0"}
 	configs[PrometheusToSd] = Config{e2eRegistry, "prometheus-to-sd", "v0.5.0"}
-	configs[Redis] = Config{e2eRegistry, "redis", "1.0"}
+	configs[Redis] = Config{dockerLibraryRegistry, "redis", "3.2.9-alpine"}
```

Comment:
all of the images need to be in `e2eRegistry` so airgapped testing can be done :sweat_smile:

Reply:
er sorry, seems that's not true for other images! we should figure that out properly someday but not this pr...
resolving